apps/ejabberd/src/mongoose_commands.erl

```diff
@@ -108,14 +108,15 @@
 
 -record(mongoose_command, {
           name :: atom(),                             %% name of the command by which we refer to it
-          category :: atom(),                         %% very important
+          category :: atom(),                         %% groups commands releated to the same functionality (user managment, messages/archive)
+          subcategory = undefined :: undefined | binary(),          %% optimal subcategory
           desc :: string(),                           %% long description
           module :: module(),                         %% module to call
           function :: atom(),                         %% function to call
-          action :: action(),                 %% so that the HTTP side can decide which verb to require
+          action :: action(),                         %% so that the HTTP side can decide which verb to require
           args = [] :: [argspec()],                   %% this is both for introspection and type check on call
           caller_pos :: integer(),                    %% internal use
-          identifiers = [] :: [atom()],               %% if action is 'update' then it must be a subset of args
+          identifiers = [] :: [atom()],               %% resource identifiers, a subset of args
           security_policy = [admin] :: security(),    %% permissions required to run this command
           result :: argspec()|ok                      %% what the called func should return; if ok then return of called
```

Comment:
In fact I personally think that all the methods should always return `ok` or `{ok, STH_MORE}`. And the decisions whether or not ignore this should be left to the caller. But that's I think a candidate for separate PR.